CHANGELOG.rst

```diff
@@ -2,30 +2,40 @@
 Changelog
```

Comment:
[NP/meta]
Is there a particular reason why we went with rST for this file?
I like rST, but it feels a bit out of place when have mostly MD in this repo.

Reply:
historical artifact; we could reformat as part of a future doc update.

Reply:
Or just do it...converted to markdown.